demo/assets/styles/app.scss

```diff
@@ -1,10 +1,15 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
 @import "~tom-select/src/scss/tom-select";
-@import '@elao/admin';
+
+//@import '@elao/admin'; // Base, built-in Design System styles
```

Comment:
Décommenter cette ligne et commenter la suivante pour switcher de la version customizée à la version pré-build du design system.

➜ A faire pour s'assurer que le dist fourni dans le package fonctionne bien out-of-the-box et inclue bien toutes les classes CSS, dont les composants à déclarer correctement dans la safelist de `tailwind.config.js`